### ISSUE TITLE
Send all option - amount not resetting after tap/untap

### ISSUE THREAD

**Issue by kiwiidb** [ACINQ/phoenix#148]
When tapping the "Send all" option, and then untapping it again, the original amount is not changed back in the input field.

Also, but this might be more of a UX issue, I find that the "send all" option is too prominent, and is easily misinterpreted by users (they sometimes think "send all" means "send all what is currently displayed on my screen").

**Comment by dpad85** [comment]
Good point, we could easily reset the amount to the invoice's initial amount (if there's one).

Regarding the "send all" checkbox prominence, I agree but I have not found a good solution. 

@robbiehanson What do you think? We'll have the same issue in the iOS app.
